src/python/WMCore/Services/WorkQueue/WorkQueue.py

```diff
@@ -61,7 +61,7 @@ def getJobStatusByRequest(self):
         data = self.db.loadView('WorkQueue', 'jobStatusByRequest',
                                 self.defaultOptions)
         return [{'request_name' : x['key'][0], 'status': x['key'][1],
-                 'jobs' : x['value']} for x in data.get('rows', [])]
+                 'jobs' : x['value']['sum']} for x in data.get('rows', [])]
```

Comment:
Since we have all the stats, I would make a keyword arg for this method (e.g. fullStats=False) such that one could either retrieve number of jobs per request+status OR stats per request+status. e.g.:
`[{'request_name' : x['key'][0], 'status': x['key'][1], 'jobs' : x['value']['sum']}]`
or
`{'request_name' : x['key'][0], 'status': x['key'][1], 'stats' : x['value']}`